Assets/MixedRealityToolkit/Boundary/Scripts/InscribedRectangle.cs

```diff
@@ -0,0 +1,570 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MixedRealityToolkit.Boundary
+{
+    /// <summary>
+    /// Helper class to hold an edge
+    /// </summary>
+    internal struct Edge
```

Comment:
Do we know for sure that this should be internal to the MRTK?
What if someone wants to utilize this struct in their own code?

Reply:
Agreed, we don't use internal (yet) in the MRTK, should be public or private to be clear.